Source/OxyPlot/Rendering/RenderContext/RenderingExtensions.cs

```diff
@@ -93,8 +95,8 @@ public static EdgeRenderingMode GetActual(this EdgeRenderingMode edgeRenderingMo
             EdgeRenderingMode edgeRenderingMode,
             double[] dashArray,
```

Comment:
`dashArray` should be nullable (`GetDashArray` in `LineStyleHelper` can return `null`)